pkg/genevalogging/bootstraplogging.go

```diff
@@ -0,0 +1,52 @@
+package genevalogging
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+import (
+	"fmt"
+
+	"github.com/Azure/go-autorest/autorest/azure"
+	"github.com/openshift/installer/pkg/asset/bootstraplogging"
+
+	"github.com/Azure/ARO-RP/pkg/api"
+	"github.com/Azure/ARO-RP/pkg/env"
+	"github.com/Azure/ARO-RP/pkg/util/tls"
+	"github.com/Azure/ARO-RP/pkg/util/version"
+)
+
+//GetBootstrapLoggingConfig prepares a bootstraplogging.Config object based on the environment
+func GetBootstrapLoggingConfig(env env.Interface, doc *api.OpenShiftClusterDocument) (*bootstraplogging.Config, error) {
+	gcskey, gcscert := env.ClustersGenevaLoggingSecret()
+
+	gcsKeyBytes, err := tls.PrivateKeyAsBytes(gcskey)
+	if err != nil {
+		return nil, err
+	}
+
+	gcsCertBytes, err := tls.CertAsBytes(gcscert)
+	if err != nil {
+		return nil, err
+	}
+
+	r, err := azure.ParseResourceID(doc.OpenShiftCluster.ID)
+	if err != nil {
+		return nil, err
+	}
+
+	return &bootstraplogging.Config{
+		Certificate:       string(gcsCertBytes),
+		Key:               string(gcsKeyBytes),
+		Namespace:         version.GenevaClusterLogsNamespace,
+		Environment:       env.ClustersGenevaLoggingEnvironment(),
+		ConfigVersion:     env.ClustersGenevaLoggingConfigVersion(),
+		Region:            env.Location(),
+		ResourceID:        doc.OpenShiftCluster.ID,
+		SubscriptionID:    r.SubscriptionID,
+		ResourceName:      r.ResourceName,
+		ResourceGroupName: r.ResourceGroup,
+		//when changing these two container images, also change /pkg/genevalogging/genevalogging.go
+		MdsdImage:      fmt.Sprintf(version.MdsdImageFormat, env.ACRName()),
+		FluentbitImage: fmt.Sprintf(version.FluentbitImageFormat, env.ACRName()),
```

Comment:
there's one more little cleanup that can be done, cf. func (g *genevaLogging) fluentbitImage() string